check_license.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+text=`cat LICENSE.txt | head -1`
+
+ERROR_COUNT=0
+while read file
+do
+    head -1 ${file} | grep -q "${text}"
+    if [ $? -ne 0 ]; then
+        echo "$file is missing license header."
+        ERROR_COUNT+=1
+    fi
+done < <(git grep -l "" | grep "\.go")
```

Comment:
I wasn't able to find an option to make `git ls-files` recurse through subdirectories. Do you know of one?